deps/obs-scripting/obs-scripting-python-frontend.c

```diff
@@ -39,7 +39,7 @@ static PyObject *get_scene_names(PyObject *self, PyObject *args)
 		PyObject *py_name = PyUnicode_FromString(*name);
 		if (py_name) {
 			PyList_Append(list, py_name);
-			Py_DECREF(py_name);
+			Py_XDECREF(py_name);
```

Comment:
There's really no benefit to most of the DECREF/XDECREF changes -- XDECREF is only needed if the object might be null, but since that's already checked a few lines earlier, it ends up being redundant to use XDECREF